tests/components/prometheus/helpers.py

```diff
@@ -0,0 +1,170 @@
+"""Helper methods for prometheus tests."""
+
+
+class MetricsTestHelper:
```

Comment:
What's the purpose of this class? It seems like all of its methods can just be separate functions in this module.